cluster/ci/config/prow/boskos.yaml

```diff
@@ -3,4 +3,20 @@ resources:
 - type: aws-quota-slice
   state: free
   min-count: 120
-  max-count: 120
\ No newline at end of file
+  max-count:  120
+- type: azure4-quota-slice
+  state: free
+  min-count: 10
```

Comment:
did you explain what `min-count` is about last time?  If so, I've forgotten; can you explain it again?